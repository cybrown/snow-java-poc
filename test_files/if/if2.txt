if (a == 1) 2 else if (a == 3) 4 else 5