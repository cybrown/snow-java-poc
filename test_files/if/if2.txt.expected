<If>
    <Binary operator="==">
        <Identifier>a</Identifier>
        <LiteralInteger>1</LiteralInteger>
    </Binary>
    <LiteralInteger>2</LiteralInteger>
    <If>
        <Binary operator="==">
            <Identifier>a</Identifier>
            <LiteralInteger>3</LiteralInteger>
        </Binary>
        <LiteralInteger>4</LiteralInteger>
        <LiteralInteger>5</LiteralInteger>
    </If>
</If>
